pom.xml

```diff
@@ -413,6 +413,16 @@
         <artifactId>pinot-minion</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.pinot</groupId>
+        <artifactId>pinot-dropwizard</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.pinot</groupId>
+        <artifactId>pinot-compound-metrics</artifactId>
```

Comment:
Can you move this after the `pinot-dropwizard` which is already defined in this file?